login-workflow/docs/customization.md

```diff
@@ -74,4 +71,16 @@ In order to work correctly, custom form components that you pass into the workfl
 
 You can see a sample implementation of the custom details forms in the `/example` project.
 
-> **NOTE:**  If you are using a useEffect hook to call the `onDetailsChanged` function, you must make sure NOT to include the `onDetailsChanged` prop in your list of dependencies. This will cause an infinite update loop.
\ No newline at end of file
+> **NOTE:** If you are using a useEffect hook to call the `onDetailsChanged` function, you must make sure NOT to include the `onDetailsChanged` prop in your list of dependencies. This will cause an infinite update loop.
+
+### Custom Registration Success Screen
+
+You can customize the success screen shown at the end of the Registration flows using the `registrationSuccessScreen` prop on the `AuthUIContextProvider`. This prop gives you access to a `registrationData` object that contains a user's `AccountDetailInformation` and email:
+
+```tsx
+registrationSuccessScreen={(registrationData) => <MySuccessScreen email={registrationData.email} firstName={registrationData.accountDetails.firstName}/>
+```
+
+#### Account Already Exists
+
+In the case when a user already has an existing account, a separate success screen is shown. You can customize this screen in the same way using the `accountAlreadyExistsScreen` prop on the `AuthUIContextProvider`.
```

Comment:
This one doesn't take any registration data arguments right? Perhaps we should add another code block showing an explicit example for this too.